gcapi/gcapi.py

```diff
@@ -263,12 +280,27 @@ class ReaderStudiesAPI(APIBase):
     answers = None  # type: ReaderStudyAnswersAPI
     questions = None  # type: ReaderStudyQuestionsAPI
 
+    def detail(self, pk):
+        result = LazyDict(lambda: super().detail(pk))
```

Comment:
So now we have to call `result.data` on everything before we can use it?

Reply:
No... the `.data` think is part of the `UserDict` contract. You could call it but you can just use the `LazyDict` directly as well. It _is_ a dict. The [-> doc](https://docs.python.org/3/library/collections.html?highlight=userdict#collections.UserDict) is really short.

Reply:
I tested this against cirrus-core already btw. It is used over there. It just works.

Reply:
Added a small test case to illustrate how lazy-dict is used.